nav2_system_tests/src/error_codes/error_code_param.yaml

```diff
@@ -0,0 +1,356 @@
+amcl:
```

Comment:
Why does this file have anything other than planner/controller server params in it?

Reply:
It uses the nav2_simple_commander.

Reply:
You don't need to bring up everything using the simple navigator // wait for things to initilize since you shouldn't launch the other stuff